tests/unit/python/foglamp/plugins/south/test_http_south.py

```diff
@@ -62,17 +59,17 @@ class TestHttpSouthDeviceUnit(object):
                 }
             }
         }"""
-        with patch.object(Ingest, 'add_readings', return_value=asyncio.ensure_future(asyncio.sleep(0.1))) as mock_method1:
-            with patch.object(Ingest, 'is_available', return_value=True) as mock_method2:
-                request = mock_request(data)
+        with patch.object(Ingest, 'add_readings', return_value=asyncio.ensure_future(asyncio.sleep(0.1))):
```

Comment:
I don't understand the return value.  We would need to patch add_readings with an awaitable. Am I missing something here?

Reply:
asyncio.ensure_future(asyncio.sleep(0.1)) is an asyncio task itself. we just want to make sure its called and its an async. otherwise we shall need a wrapper to mock async.